src/lib.rs

```diff
@@ -444,43 +466,116 @@ pub struct QuicStatisticsMisc {
     pub key_update_count: u32,
 }
 
+bitfield! {
+    #[repr(C)]
+    #[derive(Serialize, Deserialize, Clone, Copy)]
+    pub struct QuicStatisticsBitfields(u32);
+    //impl Debug; // TODO Debug causing panic when fmt is called
```

Comment:
What's this about? I don't understand it.

Reply:
@nibanks I will add a more explicit comment.

Deriving Debug is useful for quicky being able to print a struct on stdout such as with `println!("{:?}", my_connection_stats);`. So `QuicStatistics` and `QuicStatisticsV2` both derive Debug. This requires that all struct contained also implement Debug.

Rust does not have a standard equivalent to C's bitfields so I had to rely on the bitfield macro to generate the equivalent bitfields within this QuicStatisticsBitfields struct. The macro allows to derive Debug for the generated struct. However, I observed a panic when printing QuicStatisticsBitfields. I considered it simpler to just manually implement Debug for the QuicStatisticsBitfields by printing the raw u32 instead of deriving pretty printing for each bit fields.